rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
+end
```

Comment:
- You are yet to implement the belongs-to relationship between the `Rental` class and the `Book` and `Person` class. Remember that a `rental` belongs to both `Book` and `person` i.e `book.rentals << self`, `person.rentals <<self`. Please fix this 🙏